README.md

```diff
@@ -115,9 +115,38 @@ SECRET_KEY_SEED=z1AZK4h5w5YZkKYEgqtcFfvSbWQ3tZ3ZFgmLsXMZsTVoeK7 ./did id create
 // same as previous example
 ```
 
+### zCaps (Authorization Capabilities)
+
+#### Creating a Root zCap
+
+To create a root zcap that allows `read` and `write` actions to the 
+`https://example.com/api` URL:
+
+```
+ZCAP_CONTROLLER_KEY_SEED=z1AZK4h5w5YZkKYEgqtcFfvSbWQ3tZ3ZFgmLsXMZsTVoeK7 \
+  ./did zcap create \
+  --controller 'did:key:z6Mkfeco2NSEPeFV3DkjNSabaCza1EoS3CmqLb1eJ5BriiaR' \
+  --url 'https://example.com/api' \
+  --allow read --allow write
+{
+  "rootCapability": {
+    "@context": [
+      "https://w3id.org/zcap/v1",
+      "https://w3id.org/security/suites/ed25519-2020/v1"
+    ],
+    "id": "urn:zcap:root:https%3A%2F%2Fexample.com%2Fapi",
+    "controller": "did:key:z6Mkfeco2NSEPeFV3DkjNSabaCza1EoS3CmqLb1eJ5BriiaR",
+    "invocationTarget": "https://example.com/api",
+    "expires": "2022-11-24T01:17:08Z",
+    "allowedAction": ["read","write"]
```

Comment:
Not strictly necessary for a root capability, but not illegal.

Reply:
@dlongley - whoa, what do you mean? Does a root capability necessarily mean "all actions"?

Reply:
@dmitrizagidulin, yes, a root capability does not have any limitations beyond those that are built into the invocation target, so it doesn't need to have them listed.

Reply:
@dlongley - that makes sense, thank you!

Reply:
@dlongley - removed `allowedAction` from root zcaps.